packages/test-helpers/src/aragon-os/apps.js

```diff
@@ -18,6 +19,10 @@ function getInstalledApp(receipt, appId, ctx) {
   return getInstalledApps(receipt, appId, ctx)[0]
 }
 
+function getNewProxyAddress(receipt) {
+  return getEventArgument(receipt, 'NewAppProxy', 'proxy')
+}
```

Comment:
Isn't `getInstalledApp` enough? Look at [this](https://github.com/aragon/aragon-apps/blob/2090df0c5703b07d28e0af9ebd7acaffadcbf3e0/apps/finance/test/finance.js#L59-L60) example